tools/codegen/gen_backend_stubs.py

```diff
@@ -112,6 +115,49 @@ def create_backend_index(backend_ops: List[str], dispatch_key: DispatchKey) -> B
 
     return ParsedExternalYaml(backend_key, autograd_key, cpp_namespace, backend_indices)
 
+def error_on_missing_kernels(
+        native_functions: Sequence[NativeFunction],
+        backend_indices: Dict[DispatchKey, BackendIndex],
+        backend_key: DispatchKey,
+        autograd_key: DispatchKey,
+        kernel_defn_file_path: str,
+) -> None:
+    try:
+        with open(kernel_defn_file_path, 'r') as f:
+            backend_defns = f.read()
+    except IOError:
+        raise AssertionError(f'Unable to read from the specified impl_path file: {kernel_defn_file_path}')
+
+    class_name: Optional[str] = backend_indices[backend_key].native_function_class_name()
+    assert class_name is not None
+
+    expected_backend_op_names: List[OperatorName] = \
+        list(backend_indices[backend_key].index.keys()) + list(backend_indices[autograd_key].index.keys())
+    expected_backend_native_funcs: List[NativeFunction] = [f for f in native_functions if f.func.name in expected_backend_op_names]
```

Comment:
you sure you want to O(n^2) this? 🚨🚨🚨 quadratic police 🚨🚨🚨

Reply:
I might be missing it - where are you seeing the quadratic?